packages/lesswrong/server/weightedList.ts

```diff
@@ -90,7 +90,7 @@ export class WeightedList
     if (typeof weight !== typeof 1) {
       throw new Error('Weight must be numeric (got ' + (weight as any).toString() + ')');
     }
-    if (weight <= 0)  {
+    if (weight < 0)  {
```

Comment:
This came up as an error at some point, I have made it consistent with the error text (let me know if it should be the other way round)